docs/vectors/sealedbox_vectors.rst

```diff
@@ -0,0 +1,84 @@
+SealedBox reference vectors
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Since libsodium's tests do not provide reference data for the SealedBox
+construction, the implementation is verified with a ``sealbox_test_vectors``
+utility program that produces and checks custom test vectors
+by making specific calls to libsodium API.
+
+To build the ``sealbox_test_vectors`` you need a ``C`` language compiler,
+a prebuilt libsodium library more recent than version 1.0.3 and the
+corresponding include headers.
+
+In a unix-like programming environment you shoud then execute:
+
+.. code-block:: bash
+
+    $ cc -o sealbox_test_vectors -l sodium sealbox_test_vectors.c
+
+The utility program can both generate list of vectors on standard
+output and check consistency of vectors pulled from standard input.
+
+Vector generation
+"""""""""""""""""
+
+When called with one or more command line arguments, ``sealbox_test_vectors``
+will generate the number of hex-encoded vectors requested as first argument,
+with the optional second and third arguments influencing the length of the
+generated messages:
+
+.. code-block:: bash
+
+    $ ./sealbox_test_vectors 1
+    XXXX...        XXXX...        XXXX...        XXXX...
+
+The second argument, if present, sets both a minimum and a maximum length
+on generated messages overriding the default 128 bytes values
+respectively with the supplied value and with twice the supplied value.
+
+The third argument, if present, sets the maximum length of generated
+messages.
+
+Vector test
+"""""""""""
+
+When called without command line arguments, ``sealbox_test_vectors``
+will parse and hex-decode the lines given as standard input and
+check if decoding the encrypted message will return the original
+message. A "OK"/"KO" tag will be appended to the input line to
+signify success/failure of the test.
+
+To check correct "round-trip" behavior, you can run ``sealbox_test_vectors``
+as a test vector generator against itself:
+
+.. code-block:: bash
+
+    $ ./sealbox_test_vectors 1 | ./sealbox_test_vectors
+    XXXX...	XXXX...	XXXX...	XXXX...	OK
+
+If you want to check the vectors distributed with PyNaCl's sources,
+if you set the environment variable ``PYNACL_BASE`` to the directory
```

Comment:
Would 

`To complete the verification of test vectors used in PyNaCl's unittests, set the environment variable...`

be more clear way to express how this all relates to unittests?